### PR TITLE
docs: Add host volume acl docs

### PR DESCRIPTION
**Pull request by endocrimes**

### CODE REVIEW

**Comment by endocrimes** [comment]
Preview is available here: https://deploy-preview-6288--nomad-website.netlify.com/guides/security/acl.html#host_volume-rules

File: website/source/guides/security/acl.html.markdown [hunk 1]
```diff
@@ -356,6 +357,46 @@ region. When replication is interrupted, the existing data is used for request p
...
+
+host_volume "prod-ca-certificates" {
+  policy = "read"
+}
+```
+
+Host volume rules are keyed to the volume names that they apply to. As with
+namespaces, you may use wildcards to reuse the same configuration across a set
+of volumes. In addition to the coarse grained policy specification, the
+host_volume stanza allows setting a more fine grained list of capabilities.
```
Comment (tgross) [inline website/source/guides/security/acl.html.markdown]:
```suggestion
`host_volume` stanza allows setting a more fine grained list of capabilities.
```

File: website/source/guides/security/acl.html.markdown [hunk 2]
```diff
@@ -356,6 +357,46 @@ region. When replication is interrupted, the existing data is used for request p
...
+This includes:
+
+- `deny` - Do not allow a user to mount a volume in any way.
+- `mount-readonly` - Only allow the user to mount the volume as `readonly`
+- `mount-readwrite` - Allow the user to mount the volume as `readonly` or `readwrite` if the host_volume configuration allows it.
```
Comment (tgross) [inline website/source/guides/security/acl.html.markdown]:
```suggestion
- `mount-readwrite` - Allow the user to mount the volume as `readonly` or `readwrite` if the `host_volume` configuration allows it.
```

File: website/source/guides/security/acl.html.markdown [hunk 3]
```diff
@@ -356,6 +357,46 @@ region. When replication is interrupted, the existing data is used for request p
 become stale. When the authoritative region is reachable, replication will resume and repair any
 inconsistency.
 
+### host_volume Rules
+
+The `host_volume` policy controls access to mounting and accessing host volumes.
```
Comment (tgross) [inline website/source/guides/security/acl.html.markdown]: Maybe we should cross-link "host volumes" to the docs for configuring host volumes? It seems like that's been done for the other rule sets above.

Comment (endocrimes) [reply website/source/guides/security/acl.html.markdown]: I'm not sure where the best place to link to would be yet - I think I'd like to punt that until we add a bit of "summary" documentation.

**Pull request merged**